### PR TITLE
Meetup integration

### PR DESCRIPTION
**Pull request by Essk**
Ok so I don't have a lot of experience with contributing to others' projects or the Github process in general so happy to make any changes you want.

- This patch adds a Meetups component which requests the data from meetups.com. 
- The component accepts props for its title & loading messages so these can be plumbed into the CMS later if you want.
- I'm using @nuxt/axios & @nuxt/proxy to deal with CORS.
- Ideally, I'd have liked to pull individual hosts' profile pics for the tile avatars but I cannot for the life of me figure out how to pass parameters to proxy module, so just using the vixens logo for now.

### CODE REVIEW

File: .nuxt/google-analytics.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import Vue from 'vue'
 import VueAnalytics from 'vue-analytics'
 
 export default async function ({ app: { router } }) {
-  const moduleOptions = {"id":"UA-65309624-3"}
-  Vue.use(VueAnalytics, Object.assign({ router }, moduleOptions))
+  const options = {"dev":true,"debug":{"sendHitTask":false},"id":"UA-65309624-3"}
```
Comment (NataliaTepluhina) [inline .nuxt/google-analytics.js]: **question**: Why do we need `dev:true` here?

File: .nuxt/utils.js [hunk 2]
```diff
@@ -393,7 +393,7 @@ function tokensToFunction(tokens) {
         continue
       }
 
-      var value = data[token.name]
+      var value = data[token.name || 'pathMatch']
```
Comment (NataliaTepluhina) [inline .nuxt/utils.js]: **question**: was this change intended?

File: nuxt.config.js [hunk 3]
```diff
@@ -1,61 +1,104 @@
 module.exports = {
   head: {
-    title: 'Vue Vixens - Workshops for Foxy People to Learn Vue.js',
+    title: "Vue Vixens - Workshops for Foxy People to Learn Vue.js",
```
Comment (NataliaTepluhina) [inline nuxt.config.js]: **nitpick**: could you please revert this (and all upcoming) changes back to single quotes?

File: nuxt.config.js [hunk 4]
```diff
@@ -1,61 +1,104 @@
...
+      {
+        id: "UA-65309624-3"
+      }
+    ],
+    [
+      "storyblok-nuxt",
+      { accessToken: "plc0fBh2no8owEeTALSN2wtt", cacheProvider: "memory" }
+    ],
+    "@nuxtjs/axios",
+    "@nuxtjs/proxy"
```
Comment (NataliaTepluhina) [inline nuxt.config.js]: **praise**: Great work utilizing Nuxt Proxy module for fighting CORS! 🎉

File: nuxt.config.js [hunk 5]
```diff
@@ -1,61 +1,104 @@
...
+          loader: "eslint-loader",
           exclude: /(node_modules)/
-        })
+        });
       }
     }
+  },
+  proxy: {
+    "/meetups": {
+      target: "https://api.meetup.com/VueVixens/events/?&sign=true"
```
Comment (NataliaTepluhina) [inline nuxt.config.js]: **comment**: I tried to pass HTTP params here as well to get `event_hosts` shown in the response, no success 😓

File: pages/index.vue [hunk 6]
```diff
@@ -28,19 +28,23 @@
       </v-container>
     </v-parallax>
     <VVAnnouncements v-if="announcements && announcements.length" :announcements="announcements"/>
+    <VVMeetups title="Meetups" loadingMessage="Checking for meetups..." laterMessage="No meetups found, check back later?"/>
```
Comment (NataliaTepluhina) [inline pages/index.vue]: **praise**: Very nice work with error and loading cases

Comment (Essk) [reply pages/index.vue]: Thanks :D

File: nuxt.config.js (hunk 5 above)
Comment (Essk) [reply nuxt.config.js]: yeah the URI to get the avatar would be https://api.meetup.com/VueVixens/events/${EVENT_ID}/hosts?&sign=true&photo-host=public but I have no idea how to pass ${EVENT_ID} to the proxy config dynamically. Another approach could be to move the api lookup & processing to the CMS and have it passed in to asyncdata as as part of the page data but I don’t know if that’s possible with Storyblok.

File: nuxt.config.js (hunk 4 above)
Comment (Essk) [reply nuxt.config.js]: Thanks :)

File: nuxt.config.js (hunk 3 above)
Comment (Essk) [reply nuxt.config.js]: single quotes are go!

File: .nuxt/utils.js (hunk 2 above)
Comment (Essk) [reply .nuxt/utils.js]: not intentional - happened with  `npm run dev`. fixed with `git checkout master -- .nuxt/utils.js `

File: .nuxt/google-analytics.js (hunk 1 above)
Comment (Essk) [reply .nuxt/google-analytics.js]: I think there may be a bug in the build process or (quite likely) I’m doing something wrong. The dev option is set to true when `npm run dev` but is not set back to false when `npm run build`. I took a clean branch off master to demonstrate:  
run dev : https://github.com/VueVixens/website/commit/69c6bf42c9f8f19ce4d7e020e8d9d4194216c99e#diff-b40bbc46e560f2890a638d70a280e011
run build: https://github.com/VueVixens/website/commit/ace87f10e286c9b5c904b060080503cf22899efe#diff-b40bbc46e560f2890a638d70a280e011

Do you know what script is rewriting that file? If this is a bug, I think that the output should be amended there for a long term fix.

For the purpose of this PR, I ran `git checkout master -- .nuxt/google-analytics.js`

Comment (NataliaTepluhina) [reply .nuxt/google-analytics.js]: I think it could be a different version of Nuxt. Not really crucial th :)

File: nuxt.config.js (hunk 5 above)
Comment (NataliaTepluhina) [reply nuxt.config.js]: Nope, unfortunately Storyblok is not working as web proxy :(

**Comment by jlooper** [comment]
Taking a look, I think it would be nicer if we could have cards marching horizontally across the screen, same format as the Upcoming Events section (just white cards with a shadow, we probably don't need the icon). - but love the user experience! well done!

**Comment by Essk** [comment]
Hi Jen, Thanks for the review!
I've made those changes, happy to incorporate anything further, if needed.
There's only the one meet up listed right now but I did check it with some dummy data and everything grids out nicely.

**Comment by jlooper** [comment]
I ran this locally and I think this is looking good. @NataliaTepluhina if you are ok with this, I'd love to have it go live.
